src/config/generated/config.ts

```diff
@@ -0,0 +1,636 @@
+// THIS FILE IS AUTOMATICALLY GENERATED – DO NOT EDIT MANUALLY.
+
+// To parse this data:
+//
+//   import { Convert, GitProxyConfig } from "./file";
+//
+//   const gitProxyConfig = Convert.toGitProxyConfig(json);
+//
+// These functions will throw an error if the JSON doesn't
+// match the expected interface, even if the JSON is valid.
+
+/**
+ * Configuration for customizing git-proxy
+ */
+export interface GitProxyConfig {
+  /**
+   * Third party APIs
+   */
+  api?: API;
+  /**
+   * List of authentication sources for API endpoints. May be empty, in which case all
```

Comment:
Looking forward to having jsDocs for all the config elements! Hopefully, this will drive us to fill in missing descriptions in the schema (that being where they come from).